api/tests/certification/complementary-certification/unit/domain/read-models/EnrolledCandidate_test.js

```diff
@@ -0,0 +1,45 @@
+import { domainBuilder, expect } from '../../../../../test-helper.js';
+import { EnrolledCandidate } from '../../../../../../src/certification/session/domain/read-models/EnrolledCandidate.js';
+
+describe('Unit | Domain | Read Models | EnrolledCandidate', function () {
+  describe('static fromCandidateAndComplementaryCertification', function () {
+    context('when no complementary certification is provided', function () {
+      it('should build an enrolled candidate', function () {
+        // given
+        const candidate = domainBuilder.buildCertificationSessionCandidate({ userId: null });
+
+        // when
+        const enrolledCandidate = EnrolledCandidate.fromCandidateAndComplementaryCertification({
+          candidate,
+        });
+
+        // then
+        expect(enrolledCandidate).to.deepEqualInstance(
+          new EnrolledCandidate({
+            id: candidate.id,
+            firstName: candidate.firstName,
+            lastName: candidate.lastName,
+            sex: candidate.sex,
+            birthPostalCode: candidate.birthPostalCode,
+            birthINSEECode: candidate.birthINSEECode,
+            birthCity: candidate.birthCity,
+            birthProvinceCode: candidate.birthProvinceCode,
+            birthCountry: candidate.birthCountry,
+            email: candidate.email,
+            resultRecipientEmail: candidate.resultRecipientEmail,
+            externalId: candidate.externalId,
+            birthdate: candidate.birthdate,
+            extraTimePercentage: candidate.extraTimePercentage,
+            isLinked: false,
+            organizationLearnerId: candidate.organizationLearnerId,
+            complementaryCertificationId: undefined,
+            complementaryCertificationLabel: undefined,
+            complementaryCertificationKey: undefined,
+            billingMode: candidate.billingMode,
+            prepaymentCode: candidate.prepaymentCode,
+          }),
```

Comment:
```suggestion
```suggestion
          new EnrolledCandidate({
           ...candidate,
            isLinked: false,
            complementaryCertificationId: undefined,
            complementaryCertificationLabel: undefined,
            complementaryCertificationKey: undefined,
           });
```
```

Reply:
Done